dataclients/kubernetes/ingress.go

```diff
@@ -366,14 +361,14 @@ func (ing *ingress) addEndpointsRule(ic ingressContext, host string, prule *defi
 	}
 
 	if ing.kubernetesEnableEastWest {
-		ewRoute := createEastWestRouteIng(ing.eastWestDomainRegexpPostfix, meta.Name, meta.Namespace, endpointsRoute)
-		ewHost := fmt.Sprintf("%s.%s.%s", meta.Name, meta.Namespace, ing.eastWestDomainRegexpPostfix)
+		ewRoute := createEastWestRouteIng(ing.kubernetesEastWestDomain, meta.Name, meta.Namespace, endpointsRoute)
+		ewHost := fmt.Sprintf("%s.%s.%s", meta.Name, meta.Namespace, ing.kubernetesEastWestDomain)
```

Comment:
Lets check if this should be a regexp in fact

Reply:
AFAIK the `ewHost` is mostly used as the key for storing the host -> eskip.Route map. and the `route.HostRegexps` is used for the generating the Host predicate.

https://github.com/zalando/skipper/blob/5676cb745da768f18b685ce7e8e3f21c86f409e7/dataclients/kubernetes/ingress.go#L61

Reply:
ewHost is eastwest host (.skipper.cluster.local)

Reply:
@AlexanderYastrebov the tests are with files in dataclients/kubernetes/testdata/ if you have ideas what to test you can try to show that we have a bug right now.

Reply:
@szuecs I think this has to do with the deprecated `enable-kubernetes-east-west`
https://github.com/zalando/skipper/blob/5676cb745da768f18b685ce7e8e3f21c86f409e7/config/config.go#L376-L379

Here is the branch with two commits https://github.com/zalando/skipper/compare/east-west-and-range-compat
- first commit enables both deprecated and range options which generates duplicate routes
- second commit uses eastWestDomian instead of regexp suffix which applies east-west-range-predicates to the route generated by deprecated option

In general it looks like deprecated options and range options should not be used together.

For the subject discussion the `hostRoutes` map has a hostname as a key and therefore `ing.kubernetesEastWestDomain`, not regexp suffix should be used as proposed.

Reply:
@AlexanderYastrebov I am not if I understand now what you suggest to do. Can you clarify this with @girishc13 please?

Reply:
> what you suggest to do

Sorry, nothing, it should be host and it is host. Before this change it was regexp suffix which was wrong.